setup.py

```diff
@@ -258,7 +259,12 @@ def finalize_options(self):
             include_dirs=[model_dir]),
         Extension('gensim.models.fasttext_inner',
             sources=['./gensim/models/fasttext_inner.c'],
-            include_dirs=[model_dir])
+            include_dirs=[model_dir]),
+        Extension('gensim.csparse.psparse',
+            sources=['gensim/csparse/psparse.pyx', 'gensim/csparse/CSParse/Source/cs_gaxpy.c'],
+            extra_compile_args=['-fopenmp', '-O3', '-ffast-math'],
+            include_dirs = [csparse_dir, csparse_dir + '/CSParse/Include/'],
```

Comment:
`os.path.join` better? (for operating systems with other file path delimiters)

Reply:
@menshikh-iv what happens when the compilation fails (e.g. because of `openmp` issues)?
Are these extensions optional, or will this be a "hard-fail"?

Reply:
An autoconf-style check or try / except compilation with fallback shouldn't prevent getting rid of the existing sparse library. It's not much faster than using * for the multiplication.

Reply:
@piskvorky I'm not sure, but probably this will be "hard-fail", need to test it